docs/website/root/manual/getting-started/run-signer-node.md

```diff
@@ -909,7 +909,7 @@ Once this delay has passed, you should be able to observe your `PoolId` listed i
 
 ### Verify your signer is registered
 
-After installing the Mithril signer, you can verify that your node is registered by checking your Mithril signer node logs.
+After installing the Mithril signer, you can verify that your node is registered by checking your Mithril signer node logs or by running the following script.
```

Comment:
To use the same style as in `### Verify your signer contributes with individual signatures`
```suggestion
After installing the Mithril signer, you can verify that your node is registered.
```

Reply:
Done ✅